web/templates/compose.hbs

```diff
@@ -12,29 +12,29 @@
     </div>
     <div id="compose_controls">
         <div id="compose_buttons">
-            <div class="new_message_button reply_button_container">
+            <div class="reply_button_container">
                 <div class="compose-reply-button-wrapper" data-reply-button-type="selected_message">
-                    <button type="button" class="button compose_reply_button"
+                    <button type="button" class="compose_reply_button"
                       id="left_bar_compose_reply_button_big">
                         {{t 'Compose message' }}
                     </button>
                 </div>
-                <button type="button" class="button compose_new_conversation_button"
+                <button type="button" class="compose_new_conversation_button"
                   id="new_conversation_button"
                   data-tooltip-template-id="new_stream_message_button_tooltip_template">
                     {{t 'Start new conversation' }}
                 </button>
             </div>
-            <span class="new_message_button mobile_button_container">
-                <button type="button" class="button rounded compose_mobile_button"
+            <span class="mobile_button_container">
+                <button type="button" class="compose_mobile_button"
                   id="left_bar_compose_mobile_button_big"
                   data-tooltip-template-id="left_bar_compose_mobile_button_tooltip_template">
                     +
                 </button>
             </span>
             {{#unless embedded }}
-            <span class="new_message_button new_direct_message_button_container">
-                <button type="button" class="button rounded compose_new_direct_message_button"
+            <span class="new_direct_message_button_container">
+                <button type="button" class="rounded compose_new_direct_message_button"
```

Comment:
It looks like you didn't clean the `rounded` class from some of these; is that intended? 

Also, I wonder if we should have a `closed_compose_button` shared class on some of these buttons for the duplicate details, like padding? Not sure, do whatever seems best.

Reply:
OK. I think we can just clean up rounded for this pass and think about sharing more settings down the line. It's not a big deal to have a few lines of duplicate CSS.

Reply:
Updated the commit to remove `rounded`--it was just that once instance, and its `border-radius` value was already accounted for.